website/blog/2021-04-12-funding-update.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Babel Funding Update"
+author: Babel Core Team
+date: 2021-04-12 0:00:00
+categories: announcements
+share_text: "Babel Funding Update"
+---
+
+In March 2018 [Henry](https://twitter.com/left_pad) started an [experiment](https://www.henryzoo.com/in-pursuit-of-open-source-part-1/): would it be possible to work full-time on Babel, entirely sustained by donations from Babel users?
+
+In November 2019, after successfully guaranteeing a salary to a maintainer for over a year, we [expanded our goal](https://babeljs.io/blog/2019/11/08/babels-funding-plans): would it also be possible to support three additional part-time maintainers ([Jùnliàng](https://twitter.com/JLHwung), [Kai](https://twitter.com/kai_cataldo) and [Nicolò](https://twitter.com/NicoloRibaudo))?
+
+Nicolò, Jùnliàng and Henry are still paid to work on Babel, and together with them there is a talented team that helps after or during their day-to-day job.
+
+<!-- truncate -->
+
+## Looking back
+
+After a few months being paid to work on Babel, Henry started receiving $11,000 USD per month. The goal was to establish the number as a baseline salary for working full-time on an open source project that seemed reasonable given the donations at the time.
+
+When we added three new paid maintainers, we couldn't afford to pay everyone the full $11,000 USD rate: Jùnliàng, Nicolò and Kai started at $2,000 USD per month, with the idea of increasing that amount when we would have received more donations.
+
+It has been more than one year since then and in the meanwhile Kai stepped down, but we still didn't have a chance to reconsider how much everyone is being paid.
+
+We have a hard time justifying the pay differences, and there isn't an obvious reason anymore for which someone should be paid more than the other team members:
+
+- We want to focus on ourselves as _people_, not as employees of a hypothetical company. We don't want to track meticulously how many hours we work, or how productive we are.
```

Comment:
> We don't want to track meticulously how many hours we work, or how productive we are.

In a free market this tracking will happen regardless if the project is doing it or not. By choosing to not track or expose success metrics of the project you are deciding to leave this evaluation up to those deciding to continue to fund, fund, or stop funding the project.

By having some rigor around success you would be able to guide the conversation and get input from others / potential stakeholders (funders) vs not having this and leaving it as an exercise to your customers.

Reply:
Thanks for your comments!

I think here we'll want to distinguish between "progress on the project" and "progress on the project made by specific people".

We don't expect everyone of us to always have the same productivity, and we want to leave us space to experiment with things that might even not end up resulting in a concrete PR. However, we should still show the overall progress that we are making, for example with a list/timeline of "things we did" and "things we plan to do".